zerver/openapi/zulip.yaml

```diff
@@ -9514,11 +9514,18 @@ paths:
   /users:
     get:
       operationId: get-users
-      summary: Get all users
+      summary: Get users
       tags: ["users"]
       description: |
-        Retrieve details on all users in the organization. Optionally
-        includes values of [custom profile fields](/help/custom-profile-fields).
+        Retrieve details on users in the organization.
+
+        The `user_ids` query parameter can be used to limit the results to
+        specific users.
```

Comment:
Should say (either here and/or in the doc of the parameter itself) that when that's absent, it will include all users.